agent/common.sh

```diff
@@ -1,8 +1,6 @@
 #!/usr/bin/env bash
 set -euxo pipefail
 
-source network.sh
```

Comment:
That source was added in https://github.com/openshift-metal3/dev-scripts/pull/1508, is it no longer necessary?

Reply:
Waiting for a reply from @cybertron (or @elfosardo), as it doesn't seem necessary. And in case it was, it'd be better in any case to source the file here (but eventually in a step script)

Reply:
Might be worth testing this case per the comment in #1508 
>> Note that this is currently causing make clean to fail in ipv6 deployments unless you explicitly set IP_STACK.

Reply:
Thanks @bfournie good idea